### PR TITLE
RI-560 Adjust to upstream changes

### PR DESCRIPTION
**Pull request by coreywright**
* RI-560 Remove installation of shade
* Track upstream change in embedded Ansible location

Issue: https://rpc-openstack.atlassian.net/browse/RI-560

### CODE REVIEW

File: playbooks/openstack-flavor-setup.yml [hunk 1]
```diff
@@ -18,9 +18,9 @@
   environment: "{{ deployment_environment_variables | default({}) }}"
   user: root
   tasks:
-    - name: Install shade
+    - name: Install OpenStack SDK
```
Comment (odyssey4me) [inline playbooks/openstack-flavor-setup.yml]: This task is unnecessary - shade/openstacksdk are installed into the runtime by OSA:

master: https://github.com/openstack/openstack-ansible/blob/master/requirements.txt#L26
rocky: https://github.com/openstack/openstack-ansible/blob/stable/rocky/requirements.txt#L26
queens: https://github.com/openstack/openstack-ansible/blob/stable/queens/requirements.txt#L12

Only pike requires this task.

File: playbooks/openstack-image-setup.yml [hunk 2]
```diff
@@ -18,9 +18,9 @@
   environment: "{{ deployment_environment_variables | default({}) }}"
   user: root
   tasks:
-    - name: Install shade
+    - name: Install OpenStack SDK
```
Comment (odyssey4me) [inline playbooks/openstack-image-setup.yml]: This task is unnecessary - shade/openstacksdk are installed into the runtime by OSA:

master: https://github.com/openstack/openstack-ansible/blob/master/requirements.txt#L26
rocky: https://github.com/openstack/openstack-ansible/blob/stable/rocky/requirements.txt#L26
queens: https://github.com/openstack/openstack-ansible/blob/stable/queens/requirements.txt#L12

Only pike requires this task.

**Comment by coreywright** [comment]
@odyssey4me,

yes, i saw that when @cloudnull initially added the playbooks that openstack-ansible did not require shade, but then eventually added it, and recently updated the requirement to openstacksdk (with the change to ansible 2.6), some it could be considered unnecessary, but...

i kept the requirement explicit because `Explicit is better than implicit.` and this entire issue can repeat itself if openstack-ansible changes its requirements and leaves these playbooks high-and-dry (ie if you have a dependency, then install it; don't expect your other requirements to indirectly install it for you).

i have no preference (personal or professional) and considered both ways, but went with the more conservative one.  i can remove the installation of the dependency if so desired.

thanks for the review!

**Comment by coreywright** [comment]
@odyssey4me, technical debt eliminated by the removal of the unnecessary python package depedency.

**Comment by coreywright** [comment]
the previous deploy checks are failing, and i expect the currently pending deploy checks will fail,  because of an additional upstream change that broke the logging's playbooks references to the embedded ansible directory.

both the current commit and a new commit are needed to pass deploy checks, so presumably both must be merged together (if deploy checks passing are to be strictly enforced for merging).

**Comment by coreywright** [comment]
most recent failures with the singular commit "RI-560 Remove installation of shade"
* [PR_rpc-openstack-master-bionic_no_artifacts-swift-deploy #73](https://rpc.jenkins.cit.rackspace.net/job/PR_rpc-openstack-master-bionic_no_artifacts-swift-deploy/73)
* [PR_rpc-openstack-master-xenial_no_artifacts-swift-deploy #513](https://rpc.jenkins.cit.rackspace.net/job/PR_rpc-openstack-master-xenial_no_artifacts-swift-deploy/513)

```
TASK [Get osquery roles] *******************************************************
fatal: [localhost]: FAILED! => {"changed": false, "cmd": "/root/ansible25/bin/ansible-galaxy install -r ansible-role-requirements.yml --roles-path=/root/ansible25/repositories/roles --ignore-errors", "msg": "[Errno 2] No such file or directory", "rc": 2}
```

**Comment by coreywright** [comment]
woah!  how harsh!  i did not dismiss @odyssey4me review! (actually, i warmly embrace it.)

i merely added another commit to address the 2nd problem that is preventing deploy checks from passing.

File: playbooks/openstack-flavor-setup.yml (hunk 1 above)
Comment (coreywright) [reply playbooks/openstack-flavor-setup.yml]: done

File: playbooks/openstack-image-setup.yml (hunk 2 above)
Comment (coreywright) [reply playbooks/openstack-image-setup.yml]: done

**Comment by odyssey4me** [comment]
@coreywright Well, once https://review.openstack.org/614210 merges that's all going to break again. :/

**Comment by coreywright** [comment]
recheck_bionic_no_artifacts_swift_deploy

**Pull request merged**